packages/web/src/components/ui/Card/CardLink.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import CardCommon, { CardCommonProps } from './CardCommon';
+import styles from './Card.module.scss';
+import { Icon } from '../Icon/Icon';
+
+const CardLink = ({ title, to }: CardCommonProps) => (
+  <CardCommon as='a' to={to} title={title} variant='link'>
```

Comment:
Here and above, props `as` and `variant` can be inferred from the value of `to`. I.e. if `to` is truthy, then component should be a link, and a plain div otherwise.